retriever/retrieve.py

```diff
@@ -1,16 +1,15 @@
 """
-    retriever.py
+    retrieve.py
     (c) J. Renero
 
     This module retrieves latest OHLCV info and updates corresponding file.
 
 """
 import sys
 from datetime import datetime
-
-from closing import closing
-from last import last
-from rt_dictionary import RTDictionary
+from retriever.closing import Closing
+from retriever.rt_dictionary import RTDictionary
+from utils.last import last
```

Comment:
Same as before: some modules need to be specified while others don't. I guess we can keep with it but make the code a bit more inconsistent. I guess that we include the proper `import` calls in the `__init__.py` file we'll be all set.

Reply:
Same case as the previous one.